src/base1/patternfly-overrides.less

```diff
@@ -713,141 +717,4 @@ th > label {
-  --pf-global--BoxShadow--md-right: 0.3125rem 0 0.625rem -0.25rem rgba(3, 3, 3, 0.25);
-  --pf-global--BoxShadow--md-left: -0.3125rem 0 0.625rem -0.25rem rgba(3, 3, 3, 0.25);
-  --pf-global--BoxShadow--md-bottom: 0 0.3125rem 0.625rem -0.25rem rgba(3, 3, 3, 0.25);
-  --pf-global--BoxShadow--md-top: 0 -0.3125rem 0.625rem -0.25rem rgba(3, 3, 3, 0.25);
-  --pf-global--BoxShadow--lg: 0 0.1875rem 0.4375rem 0.1875rem rgba(3, 3, 3, 0.13), 0 0.6875rem 1.5rem 1rem rgba(3, 3, 3, 0.12);
-  --pf-global--BoxShadow--lg-right: 0.75rem 0 0.625rem -0.25rem rgba(3, 3, 3, 0.07);
-  --pf-global--BoxShadow--lg-left: -0.75rem 0 0.625rem -0.25rem rgba(3, 3, 3, 0.07);
-  --pf-global--BoxShadow--lg-bottom: 0 0.75rem 0.625rem -0.25rem rgba(3, 3, 3, 0.07);
-  --pf-global--BoxShadow--lg-top: 0 -0.75rem 0.625rem -0.25rem rgba(3, 3, 3, 0.07);
-  --pf-global--BoxShadow--inset: inset 0 0 0.625rem 0 rgba(3, 3, 3, 0.25);
-  --pf-global--spacer--xs: 0.25rem;
-  --pf-global--spacer--sm: 0.5rem;
-  --pf-global--spacer--md: 1rem;
-  --pf-global--spacer--lg: 1.5rem;
-  --pf-global--spacer--xl: 2rem;
-  --pf-global--spacer--2xl: 3rem;
-  --pf-global--spacer--3xl: 4rem;
-  --pf-global--spacer--form-element: 0.375rem;
-  --pf-global--gutter: 1.5rem;
-  --pf-global--gutter--md: 1rem;
-  --pf-global--golden-ratio: 1.681;
-  --pf-global--ZIndex--xs: 100;
-  --pf-global--ZIndex--sm: 200;
-  --pf-global--ZIndex--md: 300;
-  --pf-global--ZIndex--lg: 400;
-  --pf-global--ZIndex--xl: 500;
-  --pf-global--ZIndex--2xl: 600;
-  --pf-global--breakpoint--xs: 0;
-  --pf-global--breakpoint--sm: 576px;
-  --pf-global--breakpoint--md: 768px;
-  --pf-global--breakpoint--lg: 992px;
-  --pf-global--breakpoint--xl: 1200px;
-  --pf-global--breakpoint--2xl: 1450px;
-  --pf-global--link--Color: #06c;
-  --pf-global--link--Color--hover: #004080;
-  --pf-global--link--Color--light: #73bcf7;
-  --pf-global--link--Color--light--hover: #2b9af3;
-  --pf-global--link--Color--dark: #06c;
-  --pf-global--link--Color--dark--hover: #004080;
-  --pf-global--link--FontWeight: 500;
-  --pf-global--link--TextDecoration: none;
-  --pf-global--link--TextDecoration--hover: underline;
-  --pf-global--BorderWidth--sm: 1px;
-  --pf-global--BorderWidth--md: 2px;
-  --pf-global--BorderWidth--lg: 3px;
-  --pf-global--BorderColor--100: #d2d2d2;
-  --pf-global--BorderColor--200: #8a8d90;
-  --pf-global--BorderColor--300: #ededed;
-  --pf-global--BorderColor--dark-100: #d2d2d2;
-  --pf-global--BorderColor--light-100: #b8bbbe;
-  --pf-global--BorderRadius--sm: 3px;
-  --pf-global--BorderRadius--lg: 30em;
-  --pf-global--icon--Color--light: #737679;
-  --pf-global--icon--Color--dark: #151515;
-  --pf-global--icon--FontSize--sm: 0.625rem;
-  --pf-global--icon--FontSize--md: 1.125rem;
-  --pf-global--icon--FontSize--lg: 1.5rem;
-  --pf-global--icon--FontSize--xl: 3.375rem;
-  --pf-global--FontFamily--sans-serif: @font-family-base;
-  --pf-global--FontFamily--monospace: @font-family-monospace;
```

Comment:
We probably still need these two lines for a little bit longer, as PF4 doesn't use RedHatText & RedHatDisplay by default yet.